NEMO/views/users.py

```diff
@@ -481,6 +481,30 @@ def user_preferences(request):
     return render(request, "users/preferences.html", dictionary)
 
 
+@login_required
+@require_http_methods(['GET', 'POST'])
```

Comment:
just use require get and remove the check for method 
```suggestion
@require_GET
```